Add a page for Aidan (#12)
compgeolab/website: people/ahernaman.md

---
title: Aidan Hernaman
date: 2020-09-01
institution: University of Liverpool
location: Liverpool, United Kingdom
position: Undergraduate
picture: misc.jpg
github: AidanHernaman
layout: person
---

> 

Reviewer: ```suggestion
```

> # About

Reviewer: ```suggestion
# About

```

> I'm a Third year geophysics(geology) undergraduate at the university of liverpool undertaking his masters project titled "Estimating the accuracy of Moho depth estimates from gravity inversion" 

Reviewer: ```suggestion
I'm a third year Geophysics (Geology) undergraduate at the University of Liverpool undertaking my Master's project titled "Estimating the accuracy of Moho depth estimates from gravity inversion".
```
